internal/plugin/plugin.go

```diff
@@ -79,6 +79,50 @@ func (cp *CaptivePortal) Apply(ra *ndp.RouterAdvertisement) error {
 	return nil
 }
 
+type PREF64 struct {
```

Comment:
Doc comments on exported identifiers please and thanks.

Reply:
Added